warp10/src/main/java/io/warp10/continuum/ingress/Ingress.java

```diff
@@ -309,7 +309,9 @@ public Ingress(KeyStore keystore, Properties props) {
     
     this.aesDataKey = this.keystore.getKey(KeyStore.AES_KAFKA_DATA);
     this.siphashDataKey = SipHashInline.getKey(this.keystore.getKey(KeyStore.SIPHASH_KAFKA_DATA));    
-    
+
+    this.sendMetadataOnDelete = Boolean.parseBoolean(props.getProperty(Configuration.INGRESS_DELETE_METADATA_INCLUDE, "true"));
```

Comment:
I'd rather have this set to false by default since it adds an overhead in the Kafka message being sent.